src/codemagic/tools/_app_store_connect/actions/publish_action.py

```diff
@@ -29,13 +31,17 @@ class PublishAction(AbstractBaseAction, metaclass=ABCMeta):
                 PublishArgument.APPLE_ID,
                 PublishArgument.APP_SPECIFIC_PASSWORD,
                 PublishArgument.SUBMIT_TO_TESTFLIGHT,
+                BuildArgument.LOCALE_OPTIONAL,
```

Comment:
This should be the one with default value.
```suggestion
                BuildArgument.LOCALE_OPTIONAL_WITH_DEFAULT,
```